### ISSUE TITLE
Uncaught TypeError: Illegal constructor

### ISSUE THREAD

**Issue by FezVrasta** [leny/react-use-storage#2]
Hi, I get the error `Uncaught TypeError: Illegal constructor` on the line `Uncaught TypeError: Illegal constructor`.

I'm using Cypress + Chrome 74

**Comment by leny** [comment]
Hi, thanks for the issue!

Can you give me the line where the code occurs?

And maybe a bit of code where you're using it?

Thanks!

**Comment by FezVrasta** [comment]
It happens here:
https://github.com/leny/react-use-storage/blob/master/src/index.js#L11

if you try to run `new EventTarget()` on chrome 60 and lower you should be able to reproduce it.

**Comment by leny** [comment]
Thanks!
I will try to look at this this week-end.

**Comment by leny** [comment]
Did you have some code to help me reproduce the error?

**Comment by FezVrasta** [comment]
You can literally write `new EventTarget()` in the console to get the error

**Comment by leny** [comment]
Yeah but not on my browser, and I didn't have anytime to test into this version of electron.

If you want, you can modify the code inside you `node_modules`: change the line `const evtTarget = new EventTarget();` by `const evtTarget = document.createElement("phony");`, then test within your application.

That should solve the problem.

If its the case, I will test on my contexts, then publish a new version, probably tomorrow.

Thanks :)

**Comment by MichaelE1** [comment]
Hi, I ran into this issue too. Building with Gatsby:

<img width="502" alt="Screen Shot 2019-10-01 at 8 24 47 pm" src="https://user-images.githubusercontent.com/21051488/65954559-9bf9c280-e489-11e9-97ca-f1c6945456a2.png">

I then tried changing the code in `node_modules` as suggested, but that didn't work for me either:

<img width="504" alt="Screen Shot 2019-10-01 at 8 25 01 pm" src="https://user-images.githubusercontent.com/21051488/65954599-b7fd6400-e489-11e9-9726-6a23032b117c.png">
